cli/run/getConfigPath.ts

```diff
@@ -33,7 +33,7 @@ export function getConfigPath(commandConfig: string | true): string {
 
 function findConfigFileNameInCwd(): string {
 	const filesInWorkingDir = new Set(readdirSync(process.cwd()));
-	for (const extension of ['mjs', 'cjs']) {
+	for (const extension of ['mjs', 'cjs', 'ts', 'coffee']) {
```

Comment:
While TypeScript is fairly wide-spread, it feels weird to me to add special handling to CoffeeScript here.